src/test/java/org/jilt/test/data/annotations/PropagatedAnnotationName.java

```diff
@@ -0,0 +1,27 @@
+package org.jilt.test.data.annotations;
+
+import org.jilt.Builder;
+import org.jilt.BuilderStyle;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+@Builder(style = BuilderStyle.STAGED_PRESERVING_ORDER)
+public class PropagatedAnnotationName {
```

Comment:
Maybe call this class `PropagatedAnnotationsFullName` instead?

```suggestion
public class PropagatedAnnotationsFullName {
```